WordPress-iOS-Shared/Core/WPStyleGuide.m

```diff
@@ -139,111 +140,120 @@ + (UIFont *)tableviewSectionHeaderFont
     return [WPFontManager openSansBoldFontOfSize:12.0];
 }
 
+
 #pragma mark - Colors
+// https://wordpress.com/design-handbook/colors/
 
-+ (UIColor *)wordPressBlue
++ (UIColor *)colorWithR:(NSInteger)red G:(NSInteger)green B:(NSInteger)blue alpha:(CGFloat)alpha
```

Comment:
I think most objective c developers are used to the division by now as it's pretty common to see the `/255.0` littered throughout Objective-C. Why not make the naming something a little more natural and closer to the original method so that way it reads more natural and muscle memory kicks in. For example:

`[UIColor colorWithRedValue:colorWithGreenValue:colorWithBlueValue:alpha:]`